src/main/java/com/deviceinsight/kafka/health/KafkaConsumingHealthIndicator.java

```diff
@@ -60,15 +66,18 @@
 
 	private KafkaCommunicationResult kafkaCommunicationResult;
 
+	@Autowired
```

Comment:
This annotation is unnecessary. The constructor should always be used to set this field. The field can be final. For clarity, I'd move it up to the line after `cache`.

Reply:
Done.